### PR TITLE
centered-aligned call-out numbers and other fixes

### PR DESCRIPTION
**Pull request by marcfishelquantil**
This upload centers all callout numbers in tables, corrects the misspelling of "Application" in  the manifest.json file, and corrects the following broken link: Accessing Portal >Navigating UI > Image Registry > Registry Projects [404]

### CODE REVIEW

**Comment by Jianghai-Quantil** [comment]
@marcfishelquantil 
The format of callout numbers doesn't look different before and after the change. You want the callout numbers to be center-aligned horizontally, right? The preview of an example page below shows that the numbers are still left-aligned.
image

In addition, I donot find any change related to fixing the broken link as you mentioned. Is your change commited?

**Comment by marcfishelquantil** [comment]
Neil, please see my comments below.

1. The format of callout numbers doesn't look different before and after the change. You want the callout numbers to be center-aligned horizontally, right?  The preview of an example page below shows that the numbers are still left-aligned.
MARC: You are correct, I want the numbers to be center-aligned horizontally, I checked the CDN360 online help, to which I applied the same formatting, and the numbers do not appear center-aligned horizontally in GitHub; however, they do appear center-aligned horizontally on doc-dev. Please let's see how these numbers look on doc-dev, and then we can circle back if necessary.
2. Broken link: I ran a git status and it appears the Accessing Portal >Navigating UI > Image Registry topic was not checked into GitHub. I am uploading it now.

File: docs/portal/registry-users/managing-registry-users.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The following figure shows the key elements on the page, and the table following
...
-| 3                        | The **Actions** drop-down list on each row has options to [update the registry user](</docs/portal/registry-users/updating-users.md>), [change the registry user's password](</docs/portal/registry-users/changing-passwords.md>), and [delete the registry user](</docs/portal/registry-users/deleting-users.md>).                                                                          |
-| 4                        | The **+ Add New User** button allows you to [add registry users](</docs/portal/registry-users/adding-users.md>).          |
\ No newline at end of file
+<p style="text-align: center;">1</p>                                                                                                       | To filter registry users, type characters in this field and then press the Enter key. All registry users that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **Reset** button.                                             |
```
Comment (Jianghai-Quantil) [inline docs/portal/registry-users/managing-registry-users.md:15]: The preview of this list shows there are errors. Please check

image

File: docs/portal/image-pull-secrets/managing-image-pull-secrets.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The following figure shows the key elements on the page, and the table following
...
-| 2                        | Each ImagePullSecret appears on its own row. Clicking an ImagePullSecret allows you to view basic information about the ImagePullSecret.                                                           |
-| 3                        | The Actions drop-down list on each row has options to [update](</docs/portal/image-pull-secrets/updating-image-pull-secret.md>) or [delete](</docs/portal/image-pull-secrets/deleting-an-image-pull-secret.md>) an ImagePullSecret.                                                           |
-| 4                        | The **+ Add New ImagePullSecret** button allows you to [add](</docs/portal/image-pull-secrets/adding-an-image-pull-secret.md>) ImagePullSecrets.                                                     |
\ No newline at end of file
+ <p style="text-align: center;">1</p>                         | To filter ImagePullSecrets, type characters in this field and then press the Enter key. All ImagePullSecrets that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **Reset** button.                                                                   |
```
Comment (Jianghai-Quantil) [inline docs/portal/image-pull-secrets/managing-image-pull-secrets.md:22]: The preview of the list shows there are errors. Please check

image

File: docs/portal/image-pull-secrets/viewing-image-pull-secret-information.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ The following procedure describes how to view information about ImagePullSecrets
 
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
-1                                                                                                       | The Basic Information area shows basic information about the ImagePullSecret.                                     |
-| 2                        | Click this button to return to the ImagePullSecrets page.                                                     |
-| 3                        | Click this button to update the ImagePullSecret.                                                           |
-| 4                        | Click this button to delete the ImagePullSecret.                                                           |
\ No newline at end of file
+<p style="text-align: center;">1</p>                                                                                                       | The Basic Information area shows basic information about the ImagePullSecret.                                     |
```
Comment (Jianghai-Quantil) [inline docs/portal/image-pull-secrets/viewing-image-pull-secret-information.md:12]: The preview shows there are errors. Please check
image

File: docs/portal/image-registry/viewing-projects.md [hunk 4]
```diff
@@ -9,9 +9,9 @@ The following procedure describes how to view information about projects. Button
...
-| 2                        | Click this button to return to the Registry Projects page.                                                             |
-| 3                        | Click this button to view the [logs](</docs/portal/image-registry/viewing-logs.md>) associated with the project.                                                          |
-| 4                        | Click this button to [delete](</docs/portal/image-registry/deleting-projects.md>) the project.      |
-| 5                        | Click this button to [update](</docs/portal/image-registry/updating-projects.md>) the project.      |
-| 6                        | The Repositories area shows repositories associated with the project. Click a number in the **Tags** column to display [detailed tag information](</docs/portal/image-registry/managing-tags-under-a-repository.md>).                                               |
\ No newline at end of file
+<p style="text-align: center;">1</p>                                                                                                       | The Basic Information area shows basic information about the project.                                             |
```
Comment (Jianghai-Quantil) [inline docs/portal/image-registry/viewing-projects.md:12]: The preview shows there are errors. Please check.
image

File: docs/portal/registry-users/viewing-users.md [hunk 5]
```diff
@@ -9,8 +9,8 @@ The following procedure describes how to view information about registry users.
...
-| 2                        | Click this button to return to the Registry Users page.                                                                 |
-| 3                        | Click this button to change the registry user's [password](</docs/portal/registry-users/changing-passwords.md>).            |
-| 4                        | Click this button to [delete](</docs/portal/registry-users/deleting-users.md>) the registry user. |
-| 5                        | Click this button to [update](</docs/portal/registry-users/updating-users.md>) the registry user.                       |
+<p style="text-align: center;">1<p>                                                                                                       | The Basic Information area shows basic information about the registry user. To [view a project](</docs/portal/image-registry/viewing-projects.md>) associated with the registry user, click a project in the table within the **Access to Projects** field.                                            |
```
Comment (Jianghai-Quantil) [inline docs/portal/registry-users/viewing-users.md:12]: The preview shows there are errors. Please check
image

**Comment by Jianghai-Quantil** [comment]
@marcfishelquantil Regarding the issue of callout numbers format, I bring up a test environment to visualize the pages on `marc-dev`, what I've seen is that the numbers are not center-aligned as expected. Something is wrong probably. When you mention "doc-dev", are you referring to a github branch or a dev environment accessible via browser? Please give some clues, and I'll compare and see if there is something different.
image

**Comment by marcfishelquantil** [comment]
Neil, although Visual Basic did not show an issue with the tables, StackEdit did. I fixed all the tables. Please confirm that my latest upload fixes these issues. Also, my reference to docdev is http://docdev.quantil.com/, where the CDN360 and ECP content resides. Thank you!

**Comment by Jianghai-Quantil** [comment]
@marcfishelquantil I  just review the changes again. Everything looks fine to me now. PR merged.

**Pull request merged**